packages/manager/modules/pci/src/projects/project/training/registries/registries.module.js

```diff
@@ -6,13 +6,14 @@ import '@uirouter/angularjs';
 import 'angular-translate';
 import 'ovh-api-services';
 
+import component from './registries.component';
+import routing from './registries.routing';
+import service from '../registry.service';
+
 import add from './add';
-import sync from './sync';
-import component from './data.component';
-import routing from './data.routing';
-import service from '../data.service';
+import deleteC from './delete';
```

Comment:
`C` stands for what?
Could you please name your import with something more explicit?

Reply:
delete is a keyword